geoapps/inversion/components/meshes.py

```diff
@@ -142,7 +144,22 @@ def collect_mesh_params(self, params: BaseParams) -> OctreeParams:
 
     def build_from_params(self) -> Octree:
         """Runs geoapps.create.OctreeMesh to create mesh from params."""
-        octree_params = self.collect_mesh_params(self.params)
-        driver = OctreeDriver(octree_params)
-        self.entity = driver.run()
-        self.entity.parent = self.params.ga_group
+        if self.params.inversion_type in ["direct current 2d"]:
+            self.entity, self.mesh, self.permutation = get_drape_model(
+                self.workspace,
+                "Models",
+                self.inversion_data._survey.unique_locations,
+                [self.params.u_cell_size, self.params.v_cell_size],
+                self.params.depth_core,
+                [self.params.horizontal_padding] * 2
+                + [self.params.vertical_padding] * 2,
```

Comment:
This would pad upward too? I don't think we want padding in air

Reply:
right, I think i put that in there while debugging to verify the active cells and forgot to remove.  Replaced with [vertical, 1]